dexter/src/main/java/com/karumi/dexter/MultiplePermissionsReport.java

```diff
@@ -18,34 +18,37 @@
 
 import com.karumi.dexter.listener.PermissionDeniedResponse;
 import com.karumi.dexter.listener.PermissionGrantedResponse;
+
+import java.util.LinkedHashSet;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Set;
 
 /**
  * An in detail report of the request permission process
  */
 public final class MultiplePermissionsReport {
 
-  private final List<PermissionGrantedResponse> grantedPermissionResponses;
-  private final List<PermissionDeniedResponse> deniedPermissionResponses;
+  private final Set<PermissionGrantedResponse> grantedPermissionResponses;
+  private final Set<PermissionDeniedResponse> deniedPermissionResponses;
 
   MultiplePermissionsReport() {
-    grantedPermissionResponses = new LinkedList<>();
-    deniedPermissionResponses = new LinkedList<>();
+    grantedPermissionResponses = new LinkedHashSet<>();
+    deniedPermissionResponses = new LinkedHashSet<>();
```

Comment:
my  only concern here is the LinkedHashSet keep the order. If doesn't maintain the order, I'd add a comment on the get method.

Reply:
I used this one just in case the order was relevant to any user. For us it is not 😃